src/baseline_bots/bots/smart_order_accepter_bot.py

```diff
@@ -287,6 +290,102 @@ def generate_support_proposals(self, comms_obj: MessagesData) -> Dict[str, str]:
                 final_messages[recipient] = str(suggested_proposals)
                 comms_obj.add_message(recipient, str(suggested_proposals))
         return final_messages
+        
+    def is_order_aggressive_to_powers(self, order: str, powers: List[str]):
```

Comment:
It would be great to see a test for this function

Reply:
I have a test for the whole process. Will later add the test case!